package.json

```diff
@@ -144,6 +144,11 @@
     "jsSrcsDir": "./src/specs",
     "android": {
       "javaPackageName": "com.swmansion.gesturehandler"
+    },
+    "ios": {
+      "componentProvider": {
+        "RNGestureHandlerButton": "RNGestureHandlerButtonComponentView"
```

Comment:
Is this: https://github.com/software-mansion/react-native-gesture-handler/blob/6a7a128f600948e51e4c3b5eb9c4c71609666361/apple/RNGestureHandlerButtonComponentView.mm#L33-L37

still needed?

Reply:
I'm not certain, but even if it doesn't, `ios.componentProvider` is only available starting from RN 77. Therefore, to maintain backward compatibility, we still need to leave it there.

Reply:
Ah, I knew I must've forgotten something 😆